packages/manager/apps/dedicated/client/app/dedicated/server/installation/gabarit/dedicated-server-installation-gabarit.controller.js

```diff
@@ -165,20 +167,24 @@ angular
           validForm: true,
         };
 
-        Server.getHighestPriorityPartitionScheme(
+        Server.getPartitionSchemesByPriority(
           $stateParams.productId,
           $scope.installation.selectGabarit.id,
         )
+          .then((response) => {
+            $scope.installation.partitionSchemesList = response;
```

Comment:
Won't response be undefined in the next then if you don't return anything in this one ?

Reply:
Indeed the processing continue but the parameter of the next then is the value you return in the current one, in this case undefined (since nothing is returned)